Cargo.toml

```diff
@@ -8,8 +8,11 @@ license = "MIT/Apache-2.0"
 pdb = "0.3.0"
 scroll = "0.9.2"
 uuid = "0.7.2"
-wasm-bindgen = "0.2.47"
 js-sys = "0.3.24"
+wasm-bindgen = {version = "0.2.47", features = ["serde-serialize"] }
```

Comment:
Let's bump this to 0.2.48 because that's the version you've been building with. (That version difference is the main reason why there are so many lines changed in the `profiler_get_symbols.js` file, I think.)